### DISCUSSION TITLE
네이밍

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/block-it#5] [Show and tell]
- [C# naming](https://docs.popekim.com/ko/coding-standards/csharp)

**Comment by Lee-Hyeong-Jin** [comment]
C#네이밍 규칙 사용하는 것으로 통일 합시다.

**Comment by seong0929** [comment]
[유니티 C# 스타일 가이드](https://resources.unity.com/games/create-code-style-guide-e-book)

**Comment by seong0929** [comment]
[Jinger's c# naming](https://jinger.tistory.com/entry/C-%EB%84%A4%EC%9D%B4%EB%B0%8D)
